include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -317,6 +317,8 @@ struct OrtKernelContext;
 typedef struct OrtKernelContext OrtKernelContext;
 struct OrtCustomOp;
 typedef struct OrtCustomOp OrtCustomOp;
+typedef void* OrtOp;
```

Comment:
Isn't `OrtOp` the same as ORT's internal `OpKernel` ?

Shouldn't this be adhered to from the C API Guidelines ?

3. Public APIs that create/destroy an instance of an object must be declared using an established pattern and signature
If an API such as CreateSession creates an Ort object such as Session, Session class must be declared using ORT_RUNTIME_CLASS macro. The API must supply an entry point that destroys the instance of such an object. The entry point must be declared using ORT_CLASS_RELEASE and must return void.

Reply:
Done.